tasks/src/main/resources/application.conf

```diff
@@ -46,3 +46,14 @@ dataRetention {
   ortRunDays = 90
   ortRunDays = ${?DATA_RETENTION_ORT_RUN_DAYS}
 }
+
+orphanHandlers {
+  vcsInfo.limit = 1024
+  vcsInfo.limit = ${?ORPHANED_VCS_INFO_LIMIT}
+  vcsInfo.chunkSize = 64
+  vcsInfo.chunkSize = ${?ORPHANED_VCS_INFO_CHUNK_SIZE}
+  remoteArtifacts.limit = 1024
+  remoteArtifacts.limit = ${?ORPHANED_REMOTE_ARTIFACTS_LIMIT}
+  remoteArtifacts.chunkSize = 64
+  remoteArtifacts.chunkSize = ${?ORPHANED_REMOTE_ARTIFACTS_CHUNK_SIZE}
+}
```

Comment:
Do we really need to make this configurable?

Reply:
Yes, this has a high impact on the execution time of the task. I also expect that we might experiment with these values. For instance, start with a higher limit, so that the orphans that have piled up are reduced more quickly; and then adapt the value to the number of orphans that are likely produced between two executions.